trimmer_decorator.rb.rb

```diff
@@ -0,0 +1,6 @@
+class TrimmerDecorator < Decorator
```

Comment:
- [ Required ] I think it would be nice if you import the base `Decorator` class in this file before making the `TrimmerDecorator` class to inherit it, the previous reviewer had requested this change. Please see the example below for an example of how to achieve this.

- [ Optional ] I think it would also be better if you rename this file to `trimmer_decorator.rb` and not `trimmer_decorator.rb.rb` as it is currently named. A ruby file name should only have one `.rb` extensions and not two.

## Example code
> I highly recommend this change because some features are not working in your App
```ruby
require_relative 'decorator'

class TrimmerDecorator < Decorator
  def correct_name
    original_name = @nameable.correct_name
    original_name.length > 10 ? original_name[0, 10] : original_name
  end
end

```

Reply:
done